build.rs

```diff
@@ -669,6 +670,45 @@ fn nasm(file: &Path, arch: &str, include_dir: &Path, out_dir: &Path, c_root_dir:
     run_command(c);
 }
 
+fn clang_win(file: &Path, arch: &str, include_dir: &Path, out_dir: &Path) {
```

Comment:
I think instead of this, it would be better to create two separate libraries: one for assembly code, and one for C code. Then configure the `cc::Build` for the assembly code library to force the use of clang as the C compiler for Windows AArch64. This way, we maximize the use of cc-rs's infrastructure, e.g. handling compiler options and whatnot.